components/file_attachment.jsx

```diff
@@ -49,6 +49,8 @@ export default class FileAttachment extends React.Component {
             const thumbnailUrl = getFileThumbnailUrl(fileInfo.id);
 
             Utils.loadImage(thumbnailUrl, this.handleImageLoaded);
+        } else if (fileInfo.extesnion === 'svg') {
```

Comment:
maybe a type with `extesnion`

Reply:
+1 to fix the typo 😄

Reply:
Good catch. That also meant the loading logic was broken since it should've been showing the loading indicator forever because of that typo